packages/opentelemetry-instrumentation-langchain/tests/test_chains.py

```diff
@@ -39,12 +46,8 @@ def test_sequential_chain(exporter):
     )
 
     spans = exporter.get_finished_spans()
-
```

Comment:
I'm guessing you want to keep the assertions here?

Reply:
Yes, ofc, cleaned up the PR now.